src/app/shared/comcol-forms/edit-comcol-page/comcol-role/comcol-role.component.html

```diff
@@ -0,0 +1,42 @@
+<div class="card p-2">
+
+  <div class="card-body d-flex flex-column">
+
+    <div class="d-flex flex-row justify-content-between">
+
+      <div>
+        <p>{{'community.edit.tabs.roles.' + comcolRole.name + '.name' | translate}}</p>
+      </div>
+
+      <ng-container *ngVar="group$ | async as group">
+
+          <div *ngIf="!group">
+            {{'community.edit.tabs.roles.none' | translate}}
+          </div>
+
+          <a *ngIf="group"
```

Comment:
Please use routerlinks instead of hrefs. With an href the user will have to fetch the entire html page from the server again, with a routerlink, only the relevant json.

Reply:
ok, changed this in my last commit